cmd/podman/spec.go

```diff
@@ -403,6 +401,11 @@ func createConfigToOCISpec(config *createConfig) (*spec.Spec, error) {
 	return configSpec, nil
 }
 
+const (
+	bps = iota
+	iops
+)
+
```

Comment:
Exported func should need a doc/comment? not sure why the linter didnt complain about this.

Reply:
These are not exported (Lowecase)

Reply:
Right, missed that. Not sure why it was ever exported. Fixing.

Reply:
fixed.